android/src/main/java/com/reactnativevolumemanager/VolumeManagerModule.java

```diff
@@ -193,11 +194,32 @@ public void enable(final Boolean enabled, final Boolean async) {
     // no op
   }
 
+  @ReactMethod
+  public void activateAVAudioSession(final Boolean restorePreviousSessionOnDeactivation, final Boolean runAsync) {
+    // no op
+  }
+
   @ReactMethod
   public void setCategory(final String category, final Boolean mixWithOthers) {
     // no op
   }
 
+  @ReactMethod
+  public void configureAVAudioSession(final String category,
+      final String mode,
+      final String policy,
+      final ReadableArray categoryOptions,
+      final boolean prefersNoInterruptionFromSystemAlerts,
+      final boolean prefersInterruptionOnRouteDisconnect,
+      final boolean allowHapticsAndSystemSoundsDuringRecording) {
+    // no op
+  }
+
+  @ReactMethod
+  public void getAVAudioSessionStatus() {
```

Comment:
I am really unexperienced here, but maybe we should dig a bit more and give Android some love too :D